core/rend/dx11/dx11_overlay.cpp

```diff
@@ -17,24 +17,60 @@
     along with Flycast.  If not, see <https://www.gnu.org/licenses/>.
 */
 #include "dx11_overlay.h"
+#include "cfg/option.h"
 #include "rend/osd.h"
 #ifdef LIBRETRO
 #include "vmu_xhair.h"
 #endif
 
 void DX11Overlay::draw(u32 width, u32 height, bool vmu, bool crosshair)
 {
-	RECT rect { 0, 0, (LONG)width, (LONG)height };
+	RECT rect{ 0, 0, (LONG)width, (LONG)height };
 	deviceContext->RSSetScissorRects(1, &rect);
 	if (vmu)
 	{
 #ifndef LIBRETRO
 		float vmu_padding = 8.f * settings.display.uiScale;
-		float vmu_height = 70.f * settings.display.uiScale;
+		float vmu_height = 70.f * settings.display.uiScale * config::VmuScreenSize;
 		float vmu_width = 48.f / 32.f * vmu_height;
-
-		const float blend_factor[4] = { 0.75f, 0.75f, 0.75f, 0.75f };
+		const float blend_factor[4] = { config::VmuTransparency, config::VmuTransparency, config::VmuTransparency, config::VmuTransparency };
 		deviceContext->OMSetBlendState(blendStates.getState(true, 8, 8), blend_factor, 0xffffffff);
+		// Get current mouse position
```

Comment:
This is only implemented for DX11 so it shouldn't be part of this PR. Also input polling should not be done in renderers.